build.sh

```diff
@@ -24,33 +133,28 @@ then
     exit 0
 fi
 
-echo Fetching latest ckan.exe
+# Check JSON.
+echo "Running jsonlint on the changed files"
+echo "If you get an error below you should look for syntax errors in the metadata"
 
-# fetch latest ckan.exe
-wget --quiet http://ci.ksp-ckan.org:8080/job/CKAN/lastSuccessfulBuild/artifact/ckan.exe -O ckan.exe
+jsonlint -s -v ${COMMIT_CHANGES}
 
-echo Creating a dummy KSP install
+# Create folders.
+mkdir built
+mkdir downloads_cache # TODO: Point to cache folder here instead if possible.
 
-# create a dummy KSP install
-if [ "${USER}" = "jenkins" ]
-then
-    REGISTRY_FILE=${HOME}/.mono/registry/CurrentUser/software/ckan/values.xml
-    if [ -r ${REGISTRY_FILE} ]
-    then
-        rm -f ${REGISTRY_FILE}
-    fi
-fi
+# Fetch latest ckan and netkan executable.
+echo "Fetching latest ckan.exe"
+wget --quiet http://ci.ksp-ckan.org:8080/job/CKAN/lastSuccessfulBuild/artifact/ckan.exe -O ckan.exe
+
+echo "Fetching latest netkan.exe"
+wget --quiet http://ci.ksp-ckan.org:8080/job/NetKAN/lastSuccessfulBuild/artifact/netkan.exe -O netkan.exe
 
```

Comment:
These have been changed. Probably worth setting them at the top of the script to make them easier to update.

http://ckan-travis.s3.amazonaws.com/ckan.exe
http://ckan-travis.s3.amazonaws.com/netkan.exe